### PR TITLE
Inject process block

### PR DESCRIPTION
**Pull request by dvargas92495**

### CODE REVIEW

**Comment by dvargas92495** [comment]
See https://github.com/dvargas92495/roamjs-query-builder/pull/60 for where this will get used

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "lodash": "^4.17.21",
-    "roamjs-components": "^0.74.28",
+    "roamjs-components": "^0.76.2",
```
[the remark this answers is not in the document]
Comment (mdroidian) [reply package.json:29]: `8.5.0` on my desktop.  Laptop could be different.
Most of my work is done on the desktop though.

**Comment by dvargas92495** [comment]
@mdroidian Forgot to mark as ready for review earlier

**Review by mdroidian — APPROVED** [review]
Seems good.

**Pull request merged**